source/_docs/pantheon-yml.md

```diff
@@ -111,6 +72,25 @@ Add `drush_version` to the top level of the `pantheon.yml` file to configure the
 drush_version: 8
 ```
 For more information, see [Managing Drush Versions on Pantheon](/docs/drush-versions).
+
+
+### Custom Upstreams and pantheon.upstream.yml
+Add a `pantheon.upstream.yml` file to your organization's custom upstream to set default settings for all downstream sites. Downstream sites can override the defaults at the site-level with pantheon.yml. Quicksilver workflows are not supported in pantheon.upstream.yml.
```

Comment:
Suggest preserving original text that defines the upstream config file - seems like it's a bit friendlier than this iteration. For example, downstream is used without much context which can be confusing 

I also like how the original text clearly described the difference between the two


> The `pantheon.upstream.yml` file is used to set default configurations for sites created from a Custom Upstream, while the `pantheon.yml` file is used to manage configurations for one particular site. Both configuration files utilize the same set of properties that define how your Pantheon environments will behave.
>
> When the same configuration value is defined in both files, the value from `pantheon.yml` will override the value from `pantheon.upstream.yml`.